cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityUserAndRoleManagementTest.scala

```diff
@@ -29,6 +30,7 @@ class SecurityUserAndRoleManagementTest extends DocumentingTest with QueryStatis
         |** <<administration-security-roles-show, Listing roles>>
         |** <<administration-security-roles-create, Creating roles>>
         |** <<administration-security-roles-drop, Deleting roles>>
+        |** <<administration-security-roles-alter, Altering roles>>
```

Comment:
should it be named `Modifying roles` to be consistent with the user one (`Modifying users`)?